x-pack/plugins/lens/public/editor_frame_service/embeddable/expression_wrapper.tsx

```diff
@@ -8,28 +8,23 @@ import React from 'react';
 import { I18nProvider } from '@kbn/i18n/react';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { EuiFlexGroup, EuiFlexItem, EuiText, EuiIcon } from '@elastic/eui';
-import { TimeRange, Filter, Query } from 'src/plugins/data/public';
 import {
   ExpressionRendererEvent,
   ReactExpressionRendererType,
 } from 'src/plugins/expressions/public';
+import { ExecutionContextSearch } from 'src/plugins/expressions';
 
 export interface ExpressionWrapperProps {
   ExpressionRenderer: ReactExpressionRendererType;
   expression: string | null;
-  context: {
-    timeRange?: TimeRange;
-    query?: Query;
-    filters?: Filter[];
-    lastReloadRequestTime?: number;
```

Comment:
I think you need both context and searchContext. Context can contain the lastReloadRequestTime, which is currently ignored in this PR and causing a bug.